src/main/java/org/mastodon/views/trackscheme/display/TrackSchemePanel.java

```diff
@@ -449,12 +454,13 @@ else if ( flags.entitiesAttributesChanged )
 	@Override
 	public void timepointChanged()
 	{
-		final int t = timepoint.getTimepoint();
+		final int t = timepointModel.getTimepoint();
 		if ( graphOverlay.getCurrentTimepoint() != t )
 		{
 			graphOverlay.setCurrentTimepoint( t );
 			display.repaint();
 		}
+		entitiesAttributesChanged();
```

Comment:
Thanks for this advice. I was also unsure about this line, since I did not fully understand all the different flags that currently exist.
I added fadingChanged and the corresponding flad.